test_scripts/API/PerformAudioPassThru/ideas_PAPT_new_param_audioPassThruIcon.txt

```diff
@@ -0,0 +1,56 @@
+  +./test_scripts/API/PerformAudioPassThru/001_ATF_P_PerformAudioPassThru_SUCCESS_All_Params.lua 

```

Comment:
@maria-mih, will be good to add in description that audioPassThruIcon exists in app storage folder.
Is this applicable for imageType = STATIC? 
If yes, add comment, please.
This is applicable for scripts: 001-004